src/Item.php

```diff
@@ -151,15 +151,21 @@ public function get_item_tags($namespace, $tag)
     }
 
     /**
-     * Get the base URL value from the parent feed
-     *
-     * Uses `<xml:base>`
+     * Get the base URL value.
+     * Uses `<xml:base>`, or item link, or feed base URL.
      *
      * @param array $element
      * @return string
      */
     public function get_base($element = [])
     {
+        if (!empty($element['xml_base_explicit']) && isset($element['xml_base'])) {
+            return $element['xml_base'];
+        }
+        $link = $this->get_permalink();
```

Comment:
I wonder (have not checked) if some pathological feed can create an infinite loop (`get_permalink` → `get_link` → `get_links` → `get_base`).

Reply:
I do not believe it can lead to a loop